tribe-ext-eventbrite-addl-opts/index.php

```diff
@@ -188,10 +174,17 @@ public function after_iframe() {
 		return $text;
 	}
 
-	// originally from https://theeventscalendar.com/knowledgebase/changing-the-height-of-the-eventbrite-tickets-iframe/
+	/**
+	 * Replace the iframe's inline styling height within the ticket area HTML
+	 *
+	 * @param $html
+	 * @param $event_id
+	 * @param $post_id
+	 *
+	 * @return mixed|string
```

Comment:
I spoke about this with Barry because I wasn't sure exactly the format you were recommending. After showing him https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#other-descriptions, we settled on the format of `@see 'filter_name'` but he and I agreed that removing `@param` was necessary or ideal... but I'm open to further discussion about it.

I also added the filter names in `@see` for other methods even if there wasn't a callback. Is this what you'd like to see?